pkg/ossm/bug_multiple_smcp.go

```diff
@@ -10,41 +10,44 @@ import (
 
 func cleanupMultipleSMCP() {
 	util.Log.Info("Delete the Multiple CP")
-	// util.KubeDeleteContents(meshNamespace, smmr)
-	// util.KubeDeleteContents(meshNamespace, util.RunTemplate(smcpV23_template, smcp))
+	util.KubeDeleteContents(meshNS2, smmr)
+	util.KubeDeleteContents(meshNS2, util.RunTemplate(smcpV23_template, smcp))
 	time.Sleep(time.Duration(40) * time.Second)
-	util.KubeDeleteContents(meshNamespace, util.RunTemplate(smcpV23_template_meta, smcp))
+	util.KubeDeleteContents(meshNS2, util.RunTemplate(smcpV23_template_meta, smcp))
 	time.Sleep(time.Duration(40) * time.Second)
+	istio_operator_pod, _ := util.Shell(`oc get pod -n openshift-operators | grep istio-operator | awk '{print $1}'`)
+	util.Shell(`oc delete -n openshift-operators pod %s`, istio_operator_pod)
```

Comment:
You can simplify this by using label selectors. Just run the following command: 
```
oc -n openshift-operators delete pod -l name=istio-operator
```

Reply:
will update the command, Thanks for checking